containers/EmpSponsors.ts

```diff
@@ -28,6 +28,11 @@ interface PositionQuery {
   sponsor: { id: string };
   tokensOutstanding: string;
   collateral: string;
+  pendingWithdraw: string;
```

Comment:
Note that this object only defines the shape of the Query that you are receiving back from graphql. Therefore, since you are only reading the `position.withdrawalRequestTimestamp` and `position.transferPositionRequestTimestamp`, you don't have to actually set `pendingWithdraw` or `pendingTransfer` here.

Additionally, do you use `withdrawRequestAmount`?

Reply:
I see, it just shows the structure of the query.

I do use the `withdrawRequestAmount`. it only shows up if the position has a pending withdrawal. The screenshot below shows a position with a withdrawal that has ~2 mins left on it's timer.

![image](https://user-images.githubusercontent.com/12886084/89761605-0ab69c00-daef-11ea-8373-3199d7029f61.png)